docs/src/IceNucleationParcel0D.md

```diff
@@ -77,7 +79,7 @@ Accounting for conservation of water, i.e. ``\frac{dq_v}{dt} + \frac{dq_l}{dt} +
 and rearranging the terms
 ```math
 \begin{equation}
-\frac{dS_l}{dt} = a_1 w S_l - \left(a_2 + a_3 \right) S_l \frac{dq_l}{dt} - \left(a_2 + a_4\right) S_l \frac{dq_i}{dt}
+\frac{dS_l}{dt} = a_1 w S_l - \left(a_2 + a_3 S_l \right) \frac{dq_{l,vap}}{dt} - \left(a_2 + a_4  S_l\right) \frac{dq_{i,subl}}{dt} - a_5 S_l \frac{dq_{i,fus}}{dt}
```

Comment:
Shouldn't it be ``(a_2 + a_5) S_l \frac{dq_{i,fus}}{dt}`` ?

Also, since ``a_2 = S_l / q_v`` we can actually move ``S_l`` outside of everything. I.e., lets define ``a_2 = `1 / q_v``.

Reply:
If you agree then we should also change the parcel equations...